about.html

```diff
@@ -7,13 +7,39 @@
 LXQt is a lightweight Qt desktop environment.
 </p>
 <p>
-It will not get in your way. It will not hang or slow down your system. It is focused on being a classic desktop with a modern look and feel.
-</p>
-<p>LXQt is available in most Linux and BSD distributions so you may just try it out on your regular system or in a VM.<br>
-More information about installing can be found in the <a href="https://github.com/lxqt/lxqt/wiki/">LXQt GitHub Wiki</a>.
+It will not get in your way. It will not hang or slow down your system. It is focused
+ on being a classic desktop with a modern look and feel. </p>
+
+<ul>
+<li>Modular components</li>
+<li>Powerful file manager</li>
+<li>Customize look and feel everywhere</li>
+<li>Panel(s) with many plugins and settings</li>
+<li>Window Manager agnostic</li>
+</ul>
+<br/>
+
+<h3>Applications</h3>
+
+<ul>
+<li>PcManFm-qt - File Manager</li>
+<li>Lximage-qt - Image Viewer</li>
+<li>QTerminal - Terminal Emulator</li>
+<li>Qps - Process Viewer</li>
+<li>Screengrab - Screen captures</li>
```

Comment:
captures → Capturer; found only by chance.

Reply:
I almost mentioned that too.  While not exactly wrong, it does "feel" slightly weird compared to the others.